### PR TITLE
Support byte-level NAS capacity metrics

### PR DESCRIPTION
**Pull request by iltyty**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
Change unit of node_volume_capacity_bytes_total, node_volume_capacity_bytes_used, node_volume_capacity_bytes_available to bytes (nas).

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/metric/nfs_stat_collector.go [hunk 1]
```diff
@@ -443,6 +443,19 @@ func getNfsCapacityStat(pvName string, info nfsInfo, p *nfsStatCollector) ([]str
 	return nil, errors.New("capacity metrics from storage-monitor missing or invalid")
 }
 
+func isNewInfoVersion(info *nfsCapacityInfo) bool {
+	return info.UsedSizeInBytes != 0
+}
+
+func getTotalAndUsedSize(info *nfsCapacityInfo, isNewVersion bool) (total int64, used int64) {
+	if isNewVersion {
+		total, used = info.TotalSize, info.UsedSizeInBytes
```
Comment (huww98) [inline pkg/metric/nfs_stat_collector.go:452]:
```suggestion
	if info.UsedSizeInBytes != 0 {
		total, used = info.TotalSizeInBytes, info.UsedSizeInBytes
```
Maybe we don't need the isNewVersion parameter?
We may not change the meaning of existing field, or we will break users that does not upgrade CSI.

**Review by huww98 — CHANGES_REQUESTED** [review]

**Review by huww98 — CHANGES_REQUESTED** [review]
We should also change the error message in capacityEventAlert

**Review by huww98 — APPROVED** [review]
/lgtm
/assign @mowangdk
for approval
/ok-to-test

**Comment by mowangdk** [comment]
@iltyty Does this PR have any user-facing changes or impact users?

**Comment by tydra-wang** [comment]
@iltyty Maybe you should rename the title to "Support byte-level NAS capacity metrics". It does not actually change the **unit** of the metrics.

**Comment by iltyty** [comment]
> @iltyty Maybe you should rename the title to "Support byte-level NAS capacity metrics". It does not actually change the **unit** of the metrics.

done

**Comment by iltyty** [comment]
> @iltyty Does this PR have any user-facing changes or impact users?

nope

**Pull request merged**